README.md

```diff
@@ -47,4 +47,4 @@ Why not? 🤷
 
 ## About me
 
-[Duarte O.Carmo](https://duarteocarmo.com)
+I am [Duarte O.Carmo](https://duarteocarmo.com).
```

Comment:
No really a fan of this change. Can you revert?

Reply:
After that, will merge

Reply:
ok

Reply:
All done